multiterm/src/Makefile.am

```diff
@@ -24,6 +24,8 @@ multiterm_la_SOURCES = \
 	terminal.vala
 
 multiterm_la_CFLAGS = \
+	-DGETTEXT_PACKAGE='"$(GETTEXT_PACKAGE)"' \
+	-DLOCALEDIR='"$(localedir)"' \
```

Comment:
The again, I'm a tiny be stupid as `LOCALEDIR` is already passed through `AM_CFLAGS` from *vars.build.mk`, so it's redundant here.